src/Uno.UI.Composition/Composition/Visual.skia.cs

```diff
@@ -52,7 +95,7 @@ internal int ZIndex
 	internal ShadowState? ShadowState { get; set; }
 
 	/// <summary>
-	/// Render a root visual.
+	/// Render a visual as if it's the root visual.
 	/// </summary>
 	/// <param name="surface">The surface on which this visual should be rendered.</param>
 	/// <param name="ignoreLocation">A boolean which indicates if the location of the root visual should be ignored (so it will be rendered at 0,0).</param>
```

Comment:
```suggestion
	/// <param name="ignoreLocation">A boolean that indicates if the location of the root visual should be ignored (so it will be rendered at 0,0).</param>
```